packages/components/src/styled-primitives/button/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { color, space, layout, flexbox, background, border, position, shadow }
+	from 'styled-system';
+
+export const Button = styled.button`
```

Comment:
We were going to have `additionalStyles` support for `PrimitiveButton`, I don't see it here. Am I missing something?

Reply:
Sorry I missed it, it is in `packages/components/src/primitives/button/index.js`